mlxtend/classifier/stacking_classification.py

```diff
@@ -87,18 +87,26 @@ def fit(self, X, y):
             n_features is the number of features.
         y : array-like, shape = [n_samples] or [n_samples, n_outputs]
             Target values.
+        fit_params : dict, optional
+            Parameters to pass to the fit methods of `classifiers` and
+            `meta_classifier`.
 
         Returns
         -------
         self : object
 
         """
         self.clfs_ = [clone(clf) for clf in self.classifiers]
+        self.named_clfs_ = {key: value for key, value in
+                            _name_estimators(self.clfs_)}
```

Comment:
`dict(_name_estimators(self.clfs_))` might work, but no guarantee.

Reply:
hm yeah, I think it should be equivalent indeed